openedx/features/caliper_tracking/transformers/enrollment_transformers.py

```diff
@@ -58,13 +58,28 @@ def edx_course_enrollment_mode_changed(current_event, caliper_event):
     :return: final created log
     """
     caliper_event.update({
-        'type': 'Enrollment',
-        'action': 'Mode_Changed',
-        'object': current_event['event'],
-        'course_id': current_event['context']['course_id']
+        'type': 'Event',
+        'action': 'Modified',
+        'object': {
+            'id': current_event['event']['course_id'],
+            'type': 'Membership'
+        },
+    })
+
+    caliper_event['actor'].update({
+        'type': 'Person',
+        'name': current_event['username']
+    })
+
```

Comment:
Here it is only a single key to add so you may do this as
`caliper['referrer']['type'] = 'WebPage'`